tests/test_negative.py

```diff
@@ -0,0 +1,40 @@
+import json
+import subprocess
+
+import pytest
+
+from utils.command import build_analysis_command
+
+with open("data/analysis.json", "r") as file:
+    json_data = json.load(file)
+
+# Getting first instance from JSON
+first_entity_key = next(iter(json_data))
+
+
+@pytest.mark.parametrize('app_name', [first_entity_key])
```

Comment:
Here you are parametrizing the test, which is not needed since you're using only one app.

Please use `test_skip_report`  from `test_advanced_options.py` to see how you can use a single app from the `analysis.json` for a test